app/controllers/stories_controller.rb

```diff
@@ -370,7 +370,11 @@ def set_article_json_ld
         "@id": URL.article(@article)
       },
       "url": URL.article(@article),
-      "image": ApplicationController.helpers.article_social_image_url(@article),
+      "image": [
+        ApplicationController.helpers.article_social_image_url(@article, width: 1080, height: 1080),
+        ApplicationController.helpers.article_social_image_url(@article, width: 1280, height: 720),
+        ApplicationController.helpers.article_social_image_url(@article, width: 1920, height: 1080),
```

Comment:
Is there a reason this needs to be so big?

Reply:
I was just following a JSON-LD example for this, but I am happy to try making this more efficient!

Reply:
I'm inclined to follow the example all else equal, can you link to it?

Reply:
Sure thing! it can be found here: https://developers.google.com/search/docs/data-types/article

Now that I have access to the aspect ratio code, I'm going to work on making this work. 😄

Reply:
1920x1080 just seems a bit bigger than we'd realistically _need_ to provide based on the suggestions, why don't we do an even 1600x900? It's kind of arbitrary but that would work.

Reply:
OH! I think I misunderstood you. 😅 I can make that change, sure!